scripts/ScriptUtilities.psm1

```diff
@@ -36,9 +36,12 @@ function Exit-WithFailureMessage($scriptName, $message) {
 $RepoRoot = $(Resolve-Path $PSScriptRoot\..).Path
 $SourceRoot = "$RepoRoot\src"
 $NuGetPackageRoot = "$SourceRoot\packages"
+$JsonSchemaPath = "$SourceRoot\Sarif\Schemata\Sarif.schema.json"
```

Comment:
We export a couple more variables, which, although so far they're only used in Sarif.Multitool.FunctionalTests\Update-Expected.ps1, seem generally useful.